### PR TITLE
OCPBUGS#11614: Note root permissions for etcd backup script

### PR DESCRIPTION
**Pull request by kowen-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-11614

Link to docs preview:
https://63079--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#backing-up-etcd-data_backup-etcd

QE review:
- [x] QE has approved this change.

Additional information:

N/A

### CODE REVIEW

**Comment by kowen-rh** [comment]
@geliu2016 PTAL. Thank you!

File: modules/backup-etcd.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ The `cluster-backup.sh` script is maintained as a component of the etcd Cluster
 +
 [source,terminal]
 ----
-sh-4.4# /usr/local/bin/cluster-backup.sh /home/core/assets/backup
+sh-4.4# sudo /usr/local/bin/cluster-backup.sh /home/core/assets/backup
```
Comment (stevsmit) [inline modules/backup-etcd.adoc:54]: doesn't sh-4.4# already denote being logged in as a superuser? Just a question--i'm sure that this is correct.

**Comment by stevsmit** [comment]
One comment. Otherwise, LGTM

File: modules/backup-etcd.adoc (hunk 1 above)
Comment (kowen-rh) [reply modules/backup-etcd.adoc:54]: Yes, it does -- this confused me in the original issue as well, but I assumed the reporter of the issue knew something that I didn't about accessing the host binaries after using `chroot /host` (which also requires root permissions to run within the debug shell if I understand it correctly).

I just tested this procedure myself both with and without `sudo`. It works in both cases, as `oc debug node/<name>` starts a debug shell with the user logged in as `root`.

@geliu2016, can you help me to understand why `sudo` is necessary here? Is it only required in certain circumstances?

Comment (geliu2016) [reply modules/backup-etcd.adoc:54]: @kowen-rh as I know, cluster-backup.sh can only be executed by root user as I mentioned in bug: https://issues.redhat.com/browse/OCPBUGS-11614, and it have been fixed that this command could be run by non-root user, this is design change as I understanding.

Comment (geliu2016) [reply modules/backup-etcd.adoc:54]: I think so.

**Comment by kowen-rh** [comment]
Trying a different approach to this issue, suggesting `oc debug --as-root node/<node_name>`. This removes the need to use `sudo` in the backup command and ensures that the user has the appropriate permissions to run it.

I'll request a new round of QE and peer review. :slightly_smiling_face:

**Comment by kowen-rh** [comment]
@geliu2016 Could you please verify that the steps work as written here? Thank you!

**Review by lahinson — COMMENTED** [review]
/lgtm
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**